framework/include/base/FEProblemBase.h

```diff
@@ -665,6 +666,13 @@ class FEProblemBase :
   MooseSharedPointer<MultiApp> getMultiApp(const std::string & multi_app_name);
 
   /**
+   * Execute MultiAppTransfers associate with execution flag and direction.
+   * @param type The execution flag to execute.
+   * @param direction The direction (to or from) to transfer.
+   */
+  void execMultiAppTransfers(ExecFlagType type, MultiAppTransfer::DIRECTION direction);
```

Comment:
`execTransfers()` handles non-MultiApp transfers. The current execution of the MultiAppTransfers occurs in [FEProblemBase::execMultiApps](https://github.com/idaholab/moose/blob/devel/framework/src/base/FEProblemBase.C#L2814). As such, it cannot be de-coupled from the actual execution of the MultiApps.

In the level set module I need to execute the transfers with a "custom" execution flag without executing the MultiApp. This is done to transfer mesh marker information to sub-applications whenever adaptivity occurs in the master, after this is done then I run the MultiApps.

This function is completely new and removes some duplicated code to allow this.